tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -31,6 +31,16 @@
     <PackageReference Include="log4net" Version="2.0.14" Condition="'$(TargetFramework)' == 'net6.0'" />
     <PackageReference Include="log4net.Ext.Json" Version="2.0.10.1" Condition="'$(TargetFramework)' == 'net6.0'" />
 
+    <!-- MySql.Data framework references -->
+    <PackageReference Include ="MySql.Data" Version="6.10.7" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
I think we should actually test `6.10.9` instead of `6.10.7` as this is the latest version in the 6 series.
https://www.nuget.org/packages/MySql.Data